Makefile

```diff
@@ -0,0 +1,53 @@
+include ./.env
+export $(shell sed 's/=.*//' ./.env)
+
+UI_GIT_SHA = $(shell git log -1 --pretty=format:"%H")
+
+start:
+	# creates the image
+	# creates the container
+	# runs that container
+	docker build -t local/chronograf:latest -f docker/Dockerfile.chronograf .
```

Comment:
should remove this line and change the start target to
```
start: build
     #just run it now and it'll call "make build" before it starts
```

Reply:
isn't the purpose of `make build` to build the production image while `make start` is used for development (i.e. Dockerfile.choronograf)?

Reply:
ooo i missed that appendix